openfisca_france_dotations_locales/tests/dotation_solidarite_rurale/fraction_bourg_centre.yaml

```
@@ -91,3 +91,53 @@
     dsr_exclue_fraction_bourg_centre_pfi: [False, False, True]
     dsr_exclue_fraction_bourg_centre_type_1: [False, False, True]
     dsr_eligible_fraction_bourg_centre_type_1: [True, True, False]
+
+
+- name: DSR, éligibilité à la fraction bourg centre type 2. Conditions d'éligibilité (hors exclusion) 
+  period: 2020
+  input:
+    population_dgf: [10000, 10000, 10000, 20000, 9999]
+    population_insee: [10000, 10000, 10000, 20000, 9999]
+    potentiel_financier: [10000, 10000, 10000, 10000, 9999]
+    outre_mer: [False, True, False, False, False]
+    chef_lieu_arrondissement: [False, True, True, True, True]
+    part_population_canton: [0.1, 0.1, 0.1, 0.1, 0.1]
+    population_dgf_agglomeration: [10000, 10000, 10000, 20000, 9999]
+    population_dgf_departement_agglomeration: [800000, 800000, 800000, 800000, 800000]
+    population_dgf_maximum_commune_agglomeration: [10000, 10000, 10000, 20000, 9999]
+  output:
+    dsr_eligible_fraction_bourg_centre_type_2: [False, False, True, False, False]
+
+
+- name: DSR, éligibilité à la fraction bourg centre type 2. Conditions d'exclusion au titre de l'agglomération 
+  period: 2020
+  input:
+    population_dgf: [10000, 10000, 10000, 10000, 9999]
+    population_insee: [10000, 10000, 10000, 10000, 10000]
+    potentiel_financier: [10000, 10000, 10000, 10000, 10000]
+    chef_lieu_arrondissement: [True, True, True, True, True]
+    population_dgf_agglomeration: [10000, 10000, 250000, 125000, 10000]
+    population_dgf_departement_agglomeration: [800000, 20000, 8000000, 8000000, 80000]
```

L'équation utilise une autre variable. L'employer directement dans les tests pour mettre en exergue ce qui influence le résultat ? 🤔 
```suggestion
    part_population_agglomeration_departement: [0.09, 0.1, 0.1, 0.1, 0.1]
```

---

tout à fait, je vais juste grder les mêmes valeurs qu'avant (i.e. tout le monde doit être <0.1 sauf le 2ème